### PR TITLE
Error in presence attribute

### PR DESCRIPTION
**Pull request by kleihan**
Example for conditional presence uses the wrong attribute

### CODE REVIEW

**Review by donmendelson — DISMISSED** [review]
The value "optional" is syntactically valid. You could also make a case that if order type is not Stop, then presence of StopPx should be "forbidden". The difference is that "forbidden" indicates that if you populate StopPx on a Limit order, I assume you made a mistake and I will reject the order to protect you. It might be instructive to explain that difference in the text.

**Comment by kleihan** [comment]
@donmendelson what is the actual change you are looking for? The example already has a second part using "forbidden". Is it additional text in plain English?

**Comment by donmendelson** [comment]
@kleihan you're right -- the "forbidden" example is right below it. I didn't see that from the delta. It could use a line of explanation.

**Comment by kleihan** [comment]
Added text to describe the business background of the example.
